src/test/java/LinkedList/LinkedListTest.java

```diff
@@ -59,39 +77,75 @@ public void testIncludes() {
     @Test
     public void testPrint() {
 
+//        LinkedList oneList  = new LinkedList();
+//        oneList.insert(15);
+//        oneList.insert(12);
```

Comment:
You don't seem to have any actual tests in here though... so you should fix that :(